charts/eks/values.yaml

```diff
@@ -132,15 +154,41 @@ syncer:
   priorityClassName: ""
   kubeConfigContextName: "my-vcluster"
 
+# podDisruptionBudget for vcluster's pods
+podDisruptionBudget:
+  maxUnavailable: 1
+
 # Etcd settings
 etcd:
   image: public.ecr.aws/eks-distro/etcd-io/etcd:v3.4.16-eks-1-21-8
   # The amount of replicas to run 
   replicas: 1
   # NodeSelector used
   nodeSelector: {}
-  # Affinity to apply
-  affinity: {}
+  # Affinity to apply to the vcluster statefulset
+  affinity:
+    podAntiAffinity:
+      preferredDuringSchedulingIgnoredDuringExecution:
+      # if possible avoid scheduling more than one pod on one node
+      - weight: 100
+        podAffinityTerm:
+          labelSelector:
+            matchExpressions:
+            - key: app
+              operator: In
+              values:
+              - vcluster
+          topologyKey: "kubernetes.io/hostname"
+      # if possible avoid scheduling pod onto node that is in the same zone as one or more vcluster pods are running
+      - weight: 50
+        podAffinityTerm:
+          labelSelector:
+            matchExpressions:
+            - key: app
+              operator: In
+              values:
+              - vcluster
```

Comment:
etcd has the `app` label set to `vcluster-etcd` 
https://github.com/loft-sh/vcluster/blob/61419dabee3e8ad664c968f9d170cbc61003ce22/charts/eks/templates/etcd-statefulset.yaml#L25

Reply:
And same as above, you also need release: `{{ .Release.Name }}`